proto/sifnode/ethbridge/v1/tx.proto

```diff
@@ -0,0 +1,80 @@
+syntax = "proto3";
+package sifnode.ethbridge.v1;
+
+import "gogoproto/gogo.proto";
+import "sifnode/ethbridge/v1/types.proto";
+
+option go_package = "github.com/Sifchain/sifnode/x/ethbridge/types";
+
+// Msg service for messages
+service Msg {
+  rpc Lock(MsgLock) returns (MsgLockResponse);
+  rpc Burn(MsgBurn) returns (MsgBurnResponse);
+  rpc CreateEthBridgeClaim(MsgCreateEthBridgeClaim) returns (MsgCreateEthBridgeClaimResponse);
+  rpc UpdateWhiteListValidator(MsgUpdateWhiteListValidator) returns (MsgUpdateWhiteListValidatorResponse);
+  rpc UpdateCethReceiverAccount(MsgUpdateCethReceiverAccount) returns (MsgUpdateCethReceiverAccountResponse);
+  rpc RescueCeth(MsgRescueCeth) returns (MsgRescueCethResponse);
+}
+
+// MsgLock defines a message for locking coins and triggering a related event
+message MsgLock {
+  string cosmos_sender     = 1;
+  string amount            = 2 [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable)   = false,
+    (gogoproto.moretags)   = "yaml:\"amount\""];
+  string symbol            = 3 [(gogoproto.moretags) = "yaml:\"symbol\""];
+  int64  ethereum_chain_id = 4 [(gogoproto.moretags) = "yaml:\"ethereum_chain_id\""];
+  string ethereum_receiver = 5 [(gogoproto.moretags) = "yaml:\"ethereum_receiver\""];
```

Comment:
Can we use a custom type here like we do with amount, but instead of sdk.int, use ethereum.address? cc @timlind